.rhdh/scripts/install-rhdh-catalog-source.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
+#
+#  Copyright (c) 2024 Red Hat, Inc.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+# Script to streamline installing an IIB image in an OpenShift cluster for testing.
+#
+# Requires: oc, jq
+
+set -e
+
+RED='\033[0;31m'
+NC='\033[0m'
+
+NAMESPACE_CATALOGSOURCE="openshift-marketplace"
+NAMESPACE_SUBSCRIPTION="rhdh-operator"
+OLM_CHANNEL="fast"
+
+errorf() {
+  echo -e "${RED}$1${NC}"
+}
+
+usage() {
+echo "
+This script streamlines testing IIB images by configuring an OpenShift cluster to enable it to use the specified IIB image
+in a catalog. The CatalogSource is created in the openshift-marketplace namespace,
+and is named 'operatorName-channelName', eg., rhdh-fast
+
+If IIB installation fails, see https://docs.engineering.redhat.com/display/CFC/Test and
+follow steps in section 'Adding Brew Pull Secret'
+
+Usage:
+  $0 [OPTIONS]
+
+Options:
+  --latest                     : Install from iib quay.io/rhdh/iib:latest-\$OCP_VER-\$OCP_ARCH (eg., latest-v4.14-x86_64)
+  --next                       : Install from iib quay.io/rhdh/iib:next-\$OCP_VER-\$OCP_ARCH (eg., next-v4.14-x86_64)
+  --install-operator <NAME>    : Install operator named \$NAME after creating CatalogSource
+
+Developer Hub Examples:
+  $0 \\
+  --latest --install-operator rhdh # RC release in progess (from stable branch)
+
+  $0 \\
+  --next --install-operator rhdh   # CI future release (from main branch)
+"
+}
+
+# minimum requirements
+if [[ ! $(command -v oc) ]]; then
+  errorf "Please install oc 4.10+ from an RPM or https://mirror.openshift.com/pub/openshift-v4/clients/ocp/"
+  exit 1
+fi
+if [[ ! $(command -v jq) ]]; then
+  errorf "Please install jq 1.2+ from an RPM or https://pypi.org/project/jq/"
+  exit 1
+fi
+
+
+# Check we're logged into a cluster
+if ! oc whoami > /dev/null 2>&1; then
+  errorf "Not logged into an OpenShift cluster"
+  exit 1
+fi
+
+# log into your OCP cluster before running this or you'll get null values for OCP vars!
+OCP_VER="v$(oc version -o json | jq -r '.openshiftVersion' | sed -r -e "s#([0-9]+\.[0-9]+)\..+#\1#")"
+OCP_ARCH="$(oc version -o json | jq -r '.serverVersion.platform' | sed -r -e "s#linux/##")"
+if [[ $OCP_ARCH == "amd64" ]]; then OCP_ARCH="x86_64"; fi
+# if logged in, this should return something like latest-v4.12-x86_64
+UPSTREAM_IIB="quay.io/rhdh/iib:latest-${OCP_VER}-${OCP_ARCH}";
```

Comment:
you have hardcoded the `latest` prefix here. Perhaps the instructions should mention this is the default?